apps/stress-test/webpack.config.js

```diff
@@ -0,0 +1,130 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const TerserWebpackPlugin = require('terser-webpack-plugin');
+const { GriffelCSSExtractionPlugin } = require('@griffel/webpack-extraction-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+
+const isProd = process.env.NODE_ENV === 'production';
+
+const pages = [
+  {
+    name: 'v8-simple-stress',
+    path: './src/v8/simple-stress/index.tsx',
+    template: './src/v8/simple-stress/index.html',
+    output: 'v8/simple-stress/index.html',
+  },
+  {
+    name: 'v9-simple-stress',
+    path: './src/v9/simple-stress/index.tsx',
+    template: './src/v9/simple-stress/index.html',
+    output: 'v9/simple-stress/index.html',
+  },
+  {
+    name: 'wc-simple-stress',
+    path: './src/wc/simple-stress/index.wc.ts',
+    template: './src/wc/simple-stress/index.html',
+    output: 'wc/simple-stress/index.html',
+  },
+];
+
+const config = {
+  mode: isProd ? 'production' : 'development',
+  entry: pages.reduce((config, page) => {
+    config[page.name] = page.path;
+    return config;
+  }, {}),
+  output: {
+    filename: '[name].[contenthash].bundle.js',
+    sourceMapFilename: '[name].[contenthash].map',
+    path: path.resolve(__dirname, 'dist'),
+  },
+  devtool: 'source-map',
+  resolve: {
+    extensions: ['.tsx', '.ts', '.js'],
+  },
+  module: {
+    rules: [
```

Comment:
I suggest to try there to use [.oneOf](https://webpack.js.org/configuration/module/#ruleoneof) as it could make applying rules more simple. For example:

```js
module.exports = {
  //...
  module: {
    rules: [
      {
        test: /\.(ts|tsx)$/,
        oneOf: [
          // Rules for WC
          {},
          // Rules for React
          {}
        ]
      }
    ]
  }
};
```

Reply:
That definitely looks nicer. I'll take a look.

Reply:
done!